_studio/mfx_lib/shared/src/mfx_h264_enc_common_hw.cpp

```diff
@@ -7560,17 +7558,17 @@ void MfxVideoParam::SyncVideoToCalculableParam()
 
     if (IsMvcProfile(mfx.CodecProfile))
     {
-        mfxExtMVCSeqDesc * extMvc = GetExtBuffer(*this);
-        if (extMvc->NumView)
+        mfxExtMVCSeqDesc & extMvc = GetExtBufferRef(*this);
```

Comment:
Need avoid calling "GetExtBufferRef()" inside methods of MfxVideoParam. Maybe right now (in current state of the code) this particular call is OK, but it could happen that in the future this method of MfxVideoParam will be called inside constructor when preparation of ext buffers will not be finished yet, so you'll shoot in your leg by this call.

Reply:
Ok, understand, moved it back inside MfxVideoParam